conf/gulp-tasks/bundle/bundle.js

```diff
@@ -63,7 +63,7 @@ exports.legacyBundle = function (callback, outputConfig, bundleName, locale, lib
     plugins: [
       resolve({ browser: true }),
       commonjs({
-        include: [/node_modules/]
+        include: [/node_modules/, /answers-core/]
```

Comment:
If we point to a published Core beta with your changes, can we remove this change?

Reply:
yes, it will be removed before the merge.